### ISSUE TITLE
VS 2015 not compiling after ut8 commit

### ISSUE THREAD

**Issue by gorkacortazar** [openframeworks/openFrameworks#4042]
Building the openframeworks project report the following error and fails:
```
1>..\..\..\openFrameworks\utils\ofUtils.cpp(682): error C2780: 'bool std::isspace(_Elem,const std::locale &)': expects 2 arguments - 1 provided
1>  C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\include\locale(240): note: see declaration of 'std::isspace'
```
It happens in 32 and 64 bits architectures, after commit: #84793fa1f767bebbcae187d3e71a917f6b18675a

**Comment by tgfrerer** [comment]
just tested it, i have this too on VS2105. could it be that vs2015 has a different signature for `std::isspace(c)` - it requires the locale to be given.

**Comment by arturoc** [comment]
can't take a look right now but haven't changed that at all, i think it might be a missing include or something like that

**Comment by tgfrerer** [comment]
i'm working on it - seems that the trim methods need a locale specified in the second param of `std::isspace()` on VS, will send a PR in a sec - but would be good to know if other platforms have the same signature for `std::isspace`...

**Comment by bilderbuchi** [comment]
god we need some CI server soonest, I get the (subjective and wrong) feeling lately that half the issues/PRs are about issues that crop up because some introduced change was not tested on all our platforms. I think that could be easily avoided with some CI solution... :-(

**Comment by bilderbuchi** [comment]
closed by #4044
